webcompat/issues.py

```diff
@@ -43,6 +43,31 @@
 REJECTED_BODY = '''<p>The content of this issue doesn't meet our
 <a href="https://webcompat.com/terms#acceptable-use">acceptable use</a>
 guidelines. Its original content has been deleted.</p>'''
+AUTOCLOSE_TITLE = 'Issue closed.'
+AUTOCLOSE_BODY = '''<p>Thanks for the report. We have closed this issue
+automatically as we suspect it is invalid. If we made a mistake, please
+file a new issue and try to provide more context or file a report
+<a href="https://github.com/webcompat/web-bugs/issues/new/choose">
+via our GitHub repository</a>.</p>'''
```

Comment:
While this is possible, should we advertise this possibility. I'm thinking about the data structure of these bugs which are always a bit different for `body` parsing. 💭 I don't know. I let you decide.

Reply:
Yeah, I'm not sure either :) The main goal of the message is to get them to report as authorized users (if they ever come back to the issue, that is). Since bugbug is not checking those reports, so they won't be closed. 

It also could be an anchor link to https://webcompat.com/contributors/report-bug and we can have a section "how to report as authorized GitHub user". We could update this in https://github.com/webcompat/webcompat.com/issues/3348, and then add the anchor link :)

Reply:
I've removed the link for now